src/cpx_constrs.jl

```diff
@@ -130,14 +140,15 @@ function add_rangeconstrs!(model::Model, A::CoeffMat, lb::Vector, ub::Vector)
     add_rangeconstrs_t!(model, transpose(A), lb, ub)
 end
 
-function num_constr(model::Model)
+function c_api_getnumrows(model::Model)
     ncons = @cpx_ccall(getnumrows, Cint, (
                        Ptr{Void},
                        Ptr{Void}
                        ),
                        model.env.ptr, model.lp)
     return ncons
 end
+num_constr = c_api_getnumrows # Needed by MathProgBase Interface
```

Comment:
I initially missed this, it might be better to go `num_constr(model::Model) = c_api_getnumrows(model)`